src/lib/verifier/prod.mli

```diff
@@ -1 +1,39 @@
 include Verifier_intf.S with type ledger_proof = Ledger_proof.Prod.t
+
```

Comment:
I suggest we put all changes in this file in `Verifier_intf` instead

Reply:
Sure, I can do that. I avoided that because I'd have had to implement a `dummy.ml` version of the worker interface, but maybe I shouldn't be so lazy.